ge/org.osate.ge.tests/src/org/osate/ge/tests/AgeGefBot.java

```diff
@@ -148,7 +148,7 @@ protected SWTBotGefEditor createEditor(final IEditorReference reference, final S
 		}
 
 		@Override
-		public AgeSWTBotGefEditor gefEditor(String fileName) throws WidgetNotFoundException {
+		public AgeSWTBotGefEditor gefEditor(final String fileName) throws WidgetNotFoundException {
```

Comment:
While adding final is consistent with the other graphical editor code. Try to avoid making changes not relevant to the issue when working on a branch.